apps/frontend/src/app/Header.tsx

```diff
@@ -217,37 +220,34 @@ function HeaderContent({ anchor }: { anchor: string }) {
               transition: 'background-color 0.5s ease',
               backgroundColor: 'rgba(255,255,255,0.1)',
             },
-          }}
+            '& .Mui-selected': {
```

Comment:
I think the text is supposed to be bolded as well
design doc:
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/78d6c5cf-5016-4038-971f-b0625c6c4ae9)

site:
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/359ce8f9-536c-4f1a-9f4d-8eb11b87e746)

Reply:
I did try to do that, but bolding the text makes it slightly wider, so it causes a tiny bit of layout shift

Reply:
Can you try with pseudoelement with prebolded text? I think the current design is a bit hard to notice which tab you're on even with the colorization

https://stackoverflow.com/questions/556153/inline-elements-shifting-when-made-bold-on-hover

Reply:
I've tried that, and many other solutions like fake bolding with dropshadow and manual letter-spacing. I've not gotten it to work and I don't want to spend more time on it.
also, you can't seriously argue that you can't see which tab is activated here if the text isn't bolded?

![image](https://github.com/frzyc/genshin-optimizer/assets/1754901/28073f73-2df6-4f8f-9b29-ff3b3946fafd)